litmus-portal/frontend/public/locales/en/translation.yaml

```diff
@@ -30,7 +30,7 @@ welcomeModal:
 
 sidebar:
   title: Litmus
-  version: 'Version: 1.13'
+  version: "Version: 1.13"
 
```

Comment:
Please remove double/single quotes as it is not required. Only where we have to use `:` as part of the string we require to write it in quotes. Also kindly do remove such quotes which are not required at all places in the translation file

Reply:
sure these changes took place automatically due to the editor configurations

Reply:
will revert to the original file @whitetiger1399

Reply:
Also, do the necessary changes at other places also. We can take it up as part of refactoring.

Reply:
This PR cannot include refactoring changes due to the deadlines of beta 1. Patch releases can be done @whitetiger1399 cc: @rajdas98

Reply:
Yes i have already mentioned that i am reverting to the original file @arkajyotiMukherjee PTAL